chatt-app/Login.java

```diff
@@ -0,0 +1,81 @@
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.KeyEvent;
+import java.awt.event.KeyListener;
+import java.io.IOException;
+import java.net.UnknownHostException;
+
+import javax.swing.*;
+
+
+public class Login {
+	public static void main(String[] args){
+		final JFrame login = new JFrame("Login");
+		JPanel panel = new JPanel();
+		final JTextField loginName = new JTextField(20);
+		JButton enter = new JButton("Login");
+		
+		panel.add(loginName);
+		panel.add(enter);
+		login.setSize(300, 100);
+		login.add(panel);
+		login.setVisible(true);
+		login.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+		
+		enter.addActionListener(new ActionListener(){
+
+			@Override
+			public void actionPerformed(ActionEvent e) {
+				// TODO Auto-generated method stub
+				try {
+					ChatClient client = new ChatClient(loginName.getText());

```

Comment:
There is no input validation before or after the user input, loginName.getText, is passed to the ChatClient constructor. This doesn't affect functionality, but it may be needed later.